lighthouse-core/gather/driver.js

```diff
@@ -336,22 +336,24 @@ class Driver {
   sendCommandToSession(method, sessionId, ...params) {
     const timeout = this._nextProtocolTimeout;
     this._nextProtocolTimeout = DEFAULT_PROTOCOL_TIMEOUT;
-    return new Promise(async (resolve, reject) => {
-      const asyncTimeout = setTimeout((() => {
+
```

Comment:
this change is from [`no-async-promise-executor`](https://eslint.org/docs/rules/no-async-promise-executor). Usually reasonable, but to be fair the reasoning "if an async executor function throws an error, the error will be lost" is exactly why we have the try/catch in there.

Rather than ignoring the error, though, this new form is how we already handle adding a timeout in most of the rest of the code base, so it seems reasonable to use the same style and now pass the lint check.